operator/pkg/helmreconciler/prune.go

```diff
@@ -61,11 +70,34 @@ var (
 		// Cannot currently prune CRDs because this will also wipe out user config.
 		// {Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: name.CRDStr},
 	}
+
+	nonNamespacedCPResources = []schema.GroupVersionKind{
```

Comment:
why do we need these lists?

Reply:
we need these to control what resources needed to be pruned, right? And the list is different than the existing ones

Reply:
at least make it in terms of existing list and add/remove as needed.